content/400-pulse/100-what-is-pulse.mdx

```diff
@@ -49,7 +49,7 @@ A change is made to the table we've previously subscribed to. In this case, a ne
 
 ![](/img/pulse/user-insert.png)
 
-This change can happen from anywhere, the same app that uses Prisma Client to subscribe to changes, a different application or microserivce, a database GUI or any other SQL client (like `psql`).
+This change can happen from anywhere, the same app that uses Prisma Client to subscribe to changes, a different application or microserivce, a database GUI, Prisma Studio, or any other SQL client (like `psql`).
```

Comment:
I think I would personally replace the first comma with a colon, but I'm not sure if that's even correct syntax 😆 

```suggestion
This change can happen from anywhere: the same app that uses Prisma Client to subscribe to changes, a different application or microserivce, a database GUI, Prisma Studio, or any other SQL client (like `psql`).
```